app/helpers/application_helper.rb

```diff
@@ -191,9 +191,18 @@ def flash_div *keys
     keys.collect { |key|
       if flash[key]
         if flash[key].is_a?(Array)
-          content_tag(:div, content_tag(:ul, flash[key].map { |flash_item| content_tag(:li, h(flash_item)) }.join("\n").html_safe), class: "flash #{key}")
+          content_tag(
+            :div,
```

Comment:
I don't think the line break before the first argument is necessary.

Reply:
Hound was complaining about having a multi-line function call where all arguments didn't have the same indentation.

Reply:
How aboutL
```
content_tag(:div, 
            content_tag(:ul,
                        flash[key].map do |flash_item|
                          content_tag(:li, sanitize(flash_item))
                        end.join("\n").html_safe),
            class: "flash #{key}")
```

Reply:
That's essentially what I did at first and Hound didn't like it. See https://github.com/otwcode/otwarchive/pull/4651/commits/62abf6506e400df051d76adb5183d416973b9b53. But I don't care too much how it's formatted as long as it's readable - I was just following Hound's recommendations.

Reply:
> How about:
> 
> ```
> content_tag(:div, 
>             content_tag(:ul,
>                         flash[key].map do |flash_item|
>                           content_tag(:li, sanitize(flash_item))
>                         end.join("\n").html_safe),
>             class: "flash #{key}")
> ```

If we prefer this format, I can change it back?

Reply:
Yeah, I think that's much easier to read!

Reply:
Changed it back :)